modules/microshift-kickstart-rpm-install.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-kickstart-prep.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-kickstart-rpm-install_{context}"]
+= Creating a Kickstart file for a {microshift-short} RPM installation
+
+Use the Kickstart file provided with the {microshift-short} RPM to provision an RPM-based virtual machine.
+
+.Prerequisites
+
+* You set up the {microshift-short} Kickstart file.
+* You set up the [Activation Keys](https://console.redhat.com/insights/connector/activation-keys) and organization ID to activate your Red Hat subscription.
+//Q: what else?
+
+[IMPORTANT]
+====
+The subscription must include access to the `rhocp-4.x-for-rhel-9-$(uname -m)-rpms` and `fast-datapath-for-rhel-9-$(uname -m)-rpms` RPM repositories.
+====
+
+.Procedure
+
+. Add the following variables to create an RPM Kickstart file:
+
+* The `RHSM_ORG` variable contains a Red Hat Subscription Manager organization ID for the subscription registration command in the Kickstart file.
+* The `RHSM_KEY` variable contains a Red Hat Subscription Manager activation key for the subscription registration command in the Kickstart file.
+* The `MICROSHIFT_VER` variable references the {microshift-short} version to install using the `4.y` format. The latest available `.z` version is automatically installed.
+//Q: Can a user manually set the z-stream version?
```

Comment:
Not here. The way kickstart works is that it takes the latest z-stream of the `4.y` specified in `MICROSHIFT_VER`